earth_enterprise/src/server/wsgi/wms/ogc/xml/v130/capabilities_wms.py

```diff
@@ -2335,12 +2335,12 @@ class Layer(GeneratedsSuper):
     subclass = None
     superclass = None
     def __init__(self, opaque=False, cascaded=None, fixedHeight=None, fixedWidth=None, noSubsets=False, queryable=False, Name=None, Title=None, Abstract=None, KeywordList=None, CRS=None, EX_GeographicBoundingBox=None, BoundingBox=None, Dimension=None, Attribution=None, AuthorityURL=None, Identifier=None, MetadataURL=None, DataURL=None, FeatureListURL=None, Style=None, MinScaleDenominator=None, MaxScaleDenominator=None, Layer=None):
-        self.opaque = _cast(bool, opaque)
+        self.opaque = _cast(int, opaque)
```

Comment:
What's the reason for changing the cast of opaque, noSubsets, and queryable to int?  Looking at the file they are set to bool values of True/False.

Reply:
Trying to integrate our WMS serving with OWSLib. The library expects our responses to return integers. So it's fine to set true/false throughout, but the actual value will need the cast.

Reply:
ok, that works.  Thanks for the explanation.